internal/fs/fs_test.go

```diff
@@ -140,6 +141,11 @@ func (t *fsTest) SetUpTestSuite() {
 	t.serverCfg.SequentialReadSizeMb = SequentialReadSizeMb
 	if t.serverCfg.MountConfig == nil {
 		t.serverCfg.MountConfig = config.NewMountConfig()
+	} else {
+		mountConfig := t.serverCfg.MountConfig
```

Comment:
These could have been done by making changes only in type_cache_test.go, but putting these changes here instead has the advantage that with this, any new/existing composite tests can simply set their own mountConfig now (equivalent of config-file if they need) instead of replicating this code in their test files.

Reply:
I just wanted to get ride of changes in the fs_test.SetUpTestSuite(). 

But anyway changing the DirTypeCacheTTL and InodeAttributeCacheTTL based on MountingConfig check doesn't look good.

Reply:
Please make the else code un-conditional. It should always be executed.

Reply:
Will do.